utils/src/treev2/quickPickWizard/QuickPickWizardContext.ts

```diff
@@ -5,7 +5,7 @@
 
 import * as types from '../../../index';
 
-export function getLastNode<TNode extends unknown>(context: types.QuickPickWizardContext<TNode>): TNode | undefined {
+export function getLastNode(context: types.QuickPickWizardContext): unknown | undefined {

```

Comment:
I like the idea of keeping this generic so that callers that _know_ the type of the node can pre-cast it, instead of needing to do an `as Blah`.
```suggestion
export function getLastNode<TNode = unknown>(context: types.QuickPickWizardContext): TNode | undefined {
```